src/purchases.ts

```diff
@@ -68,6 +68,38 @@ export enum PURCHASE_TYPE {
     SUBS = "subs",
 }
 
+/**
+ * Enum for billing features.
+ * Currently, these are only relevant for Google Play BillingClient:
+ * https://developer.android.com/reference/com/android/billingclient/api/BillingClient.FeatureType
+ */
+export enum BILLING_FEATURE {
+    /**
+     * https://developer.android.com/reference/com/android/billingclient/api/BillingClient.FeatureType#SUBSCRIPTIONS
+     */
+    SUBSCRIPTIONS,
```

Comment:
This documentation shows up in IDEs and I think it's better if we copy Google's docs instead of linking to Google's documentation.